client.go

```diff
@@ -392,8 +400,11 @@ func (c *Client) Kill() {
 	if graceful {
 		select {
 		case <-doneCh:
+			// FIXME: we can never reach here, because the plugin process is
+			// never signaled to exit.
```

Comment:
I believe this can happen in cases where the plugin exited on its own (probably not gracefully but on its own).

Reply:
good point, I'll reword that